### DISCUSSION TITLE
Display additional product info on product details page

### DISCUSSION THREAD

**Discussion by gimmins** [vendurehq/vendure#1406] [Q&A]
To Vendure gurus, 

How can I display additional info from Product object in the product details page? This is not a `customField`, but rather just a field that displays existing data

![Screen Shot 2022-02-08 at 12 47 15 AM](https://user-images.githubusercontent.com/2350082/152951127-738cae6b-f0d9-4cfe-a0dd-cf51eab97e02.png)

Much appreciate your help! 🙏

**Comment by michaelbromley** [comment]
Hi!
You can use a [CustomDetailComponent](https://www.vendure.io/docs/plugins/extending-the-admin-ui/custom-detail-components/) to do this.

Also note that "createdAt" can be viewed by clicking the info icon in the top left of the page.
